runtime/src/bank.rs

```diff
@@ -7389,8 +7414,9 @@ pub(crate) mod tests {
 
         let account_pubkey = solana_sdk::pubkey::new_rand();
         let account_balance = 1;
+        let data_len = 12345; // use non-zero data len to also test accounts_data_len
```

Comment:
Can you add an assert for `accounts_data_len` in this test?

Reply:
Done